test/jellyfish_web/controllers/component_controller_test.exs

```diff
@@ -9,7 +9,7 @@ defmodule JellyfishWeb.ComponentControllerTest do
     server_api_token = Application.fetch_env!(:jellyfish, :server_api_token)
     conn = put_req_header(conn, "authorization", "Bearer " <> server_api_token)
 
-    room_conn = post(conn, ~p"/room")
+    room_conn = post(conn, ~p"/room", enforcedVideoCodec: "h264")
```

Comment:
I think that h264 should be enforced only in the HLS component test, so we test other codecs as well.
You can create a setup block only for the `describe "create hls component"`